CMakeLists.txt

```diff
@@ -119,4 +127,8 @@ set(RETSIGTYPE "void")
 set(STDC_HEADERS 1)
 check_include_files("time.h;sys/time.h" TIME_WITH_SYS_TIME)
 
-configure_file(src/config.h.in src/config.h)
\ No newline at end of file
+configure_file(src/config.h.in src/config.h)
+
+message(STATUS "CXXFlags  : ${CMAKE_CXX_FLAGS}")
```

Comment:
Is there a specific reason to write `CFlags`, `CXXFlags` and `Link Flags` instead of `C++ compiler flags`, `C compiler flags` and `Linker flags`? The former resembles like a bad formatting of autotools variable names and doesn't look good to me.

Reply:
OK, I was just wondering where it came from, because I haven't seen it anywhere before.